example-apps/chatbot-rag-app/k8s-manifest.yml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: chatbot-rag-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: chatbot-rag-app
+  template:
+    metadata:
+      labels:
+        app: chatbot-rag-app
+    spec:
+      # Simplify networking so that it can look for elasticsearch on localhost
+      hostNetwork: true
```

Comment:
@anuraaga made a good point offline that if just trying to get things listening on localhost, k8s is the hardest option. Maybe focus this on normal k8s instead, even if it is more work than this PR stands so far.

* instructions to setup Elastic Stack in k8s, including APM server (maybe another directory in this repo like our "docker" one.
* Have to edit the properties in .env to make sure you use the right hostname for elasticsearch and the otel endpoint as localhost won't work anymore

Might be best to go ahead and do that I think, so will unless someone feels otherwise.